src/ServiceControl.Transports.ASBS/Helper.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ServiceControl.Transports.ASBS
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using NServiceBus;
+
+    public static class Helper
+    {
+        public static void ConfigureTransport(this TransportExtensions<AzureServiceBusTransport> transport, TransportSettings transportSettings)
+        {
+            transport.ConfigureSanitization();
+
+            transport.Transactions(TransportTransactionMode.SendsAtomicWithReceive);
+            transport.ConnectionString(transportSettings.ConnectionString);
+        }
+
+        public static void ConfigureSanitization(this TransportExtensions<AzureServiceBusTransport> transport)
+        {
+            transport.SubscriptionNameShortener(n => n.Length > MaxEntityName ? MD5DeterministicNameBuilder.Build(n) : n);
```

Comment:
I assume these are the same rules that are default for the old transport?

Reply:
The intention is to make it backward compat with forwarding in ASB yes.